### PR TITLE
Sign up requests

### PR DESCRIPTION
**Pull request by GubbyDuo**
Added LoginRequest class, granting ability for users to recieve information back when logging in. Signup also works, but worked before these changes.
Changed 'passWord' to 'password'.
Added Security override(?) for testing purposes. Postman will now work without errors.
Added custom repository method.

### CODE REVIEW

File: src/main/java/com/traveler/travelapp/loginRequest/LoginRequest.java [hunk 1]
```diff
@@ -0,0 +1,27 @@
+package com.traveler.travelapp.loginRequest;
+
+public class LoginRequest {
+    private String userName;
+    private String password;
```
Comment (SeanEngineering) [inline src/main/java/com/traveler/travelapp/loginRequest/LoginRequest.java:13]: Hey Sam, did can you please make a ticket to encrypt password

File: src/main/java/com/traveler/travelapp/user/UserController.java
Comment (SeanEngineering) [inline src/main/java/com/traveler/travelapp/user/UserController.java:1]: Please move repository calls to service layer

**Comment by SeanEngineering** [comment]
Can you implement Lombok too @GubbyDuo

**Comment by GubbyDuo** [comment]
> Can you implement Lombok too @GubbyDuo

Can do on the weekend or maybe next week

**Comment by SeanEngineering** [comment]
@GubbyDuo pinged you on teams. Did you want to do the service stuff later?

File: src/main/java/com/traveler/travelapp/loginRequest/LoginRequest.java [hunk 2]
```diff
@@ -1,27 +1,16 @@
...
 
+import jakarta.persistence.Entity;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.Setter;
+
+@Entity
+@Getter
+@Setter
+@AllArgsConstructor
```
Comment (SeanEngineering) [inline src/main/java/com/traveler/travelapp/loginRequest/LoginRequest.java:10]: You need a @NoArgsConstructor for entities too

**Review by SeanEngineering — APPROVED** [review]
Make sure you have @Autowired on services and repos when you create them.

Other than that lgtm

**Pull request merged**